utils.py

```diff
@@ -0,0 +1,36 @@
+from qtpy import QtCore, QtGui, QtWidgets
+
+
+def morph_into_vertical(label: QtWidgets.QLabel):
```

Comment:
This is a neat way of doing things. Consider also that you might use the "promoted" widget option built into Qt designer (e.g., and have a `QLabel` subclass `pmps.VerticalLabel` that implements the closure methods below).

Reply:
This is inherited code that I've refactored over to a utils submodule. Good tip about the promotion, that's a cleaner way to set it up.